plugin/trino-kafka/src/test/resources/protobuf/test_oneof.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+message schema {
+    oneof testOneOfColumn {
+        string stringColumn = 1;
+        uint32 integerColumn = 2;
+        uint64 longColumn = 3;
+        double doubleColumn = 4;
+        float floatColumn = 5;
+        bool booleanColumn = 6;
```

Comment:
Can we also pass a `message` type and check on how `DynamicMessage` would be serialized.